.circleci/config.yml

```diff
@@ -354,6 +329,26 @@ jobs:
           paths:
             - autopush-locust.tar
 
+  build-integration-test:
```

Comment:
Thank you for adding the CI stuff.
I think this job needs to be added to the workflow? Can the image be re-used by the integration test job as a pre-req?

Reply:
Yeah I didn't add it as I wanted to do it in a separate PR. It could be reused after it's pushed to GAR. That would allow us to use circleci layer caching too actually.